### ISSUE TITLE
looking for customized titles for this Report like Project Name, Release name and Cycle name

### ISSUE THREAD

**Issue by Sanj2005** [wswebcreation/protractor-multiple-cucumber-html-reporter-plugin#6]
Hi - This report is really great. I was wondering if we can have additional details on the top of the Report like below
- Project/Application Name: 
- Release Name:
- Cycle Name:
- Total execution Time:

User should be allowed to pass this data as below

```js
  plugins: [{
    package: 'protractor-multiple-cucumber-html-reporter-plugin',
    options: {
      automaticallyGenerateReport: true,
      removeExistingJsonReportFile: true,
      removeOriginalJsonReportFile: true,
      openReportInBrowser: true,
      saveCollectedJSON: true,
      jsonOutputPath: './reports/report-plugin-json-output-folder/',
      reportPath: './reports/protractor-multiple-cucumber-html-report/',
      disableLog: false,
      metadataKey: 'deviceProperties',
      Project/Application Name : 'XYZ Automation',
      Release Name: 'ABC',
      Cycle  : '2'
    }
```

Also if possible can we rename the report name - "Multiple Cucumber HTML Report" as like below
- Multi Browser Protractor Cucumber HTML Report

Thanks

**Comment by wswebcreation** [comment]
Hi @Sanj2005 

Tnx for the future request, but this isn't the right place because this is only a plugin that uses https://github.com/wswebcreation/multiple-cucumber-html-reporter. 

I like all the ideas except the execution time because that isn't saying anything, personally I'm against it because it will only tell you something about a step, but not the complete execution (spinning up the browser, loading page and so on.

You may also add a PR to https://github.com/wswebcreation/multiple-cucumber-html-reporter. For now I'm going to close it here

**Comment by Sanj2005** [comment]
Sure... Thanks wswebcreation for the quick response !!!!
I have created the enhancement request to https://github.com/wswebcreation/multiple-cucumber-html-reporter 
Just wondering if I can pass the same parameters by using this plugin as well.  
Many thanks again for putting this together !!!

**Comment by wswebcreation** [comment]
@Sanj2005 

I'll need to update this module if the report-module has been updated

**Comment by wswebcreation** [comment]
@Sanj2005 

Implemented this with version 1.3.0, see https://github.com/wswebcreation/protractor-multiple-cucumber-html-reporter-plugin/compare/v1.2.5...v1.3.0

**Comment by Sanj2005** [comment]
Yes I checked it....its looking gr8..Thanks!!!
